hudi-client/src/main/java/org/apache/hudi/table/WorkloadProfile.java

```diff
@@ -54,13 +55,23 @@
    */
   private final WorkloadStat globalStat;
 
+  /**
+   * Write operation type.
+   */
+  private WriteOperationType operationType;
```

Comment:
WriteOperationType should be Serializable?

Reply:
@leesf Yes, WriteOperationType should be Serializable, forget to check this. I would like to add `implements Serializable`.